packages/common/src/ui/layout/tables/components/Expand/MiniTable.tsx

```diff
@@ -30,7 +31,9 @@ export const MiniTable = <T extends RecordWithId>({
           overflow: 'hidden',
         }}
       >
-        <DataTable dense columns={columns} data={rows} />
+        <TableProvider createStore={createTableStore}>
```

Comment:
This was the problem. creating a new store here isolates the context to each table and fixes the issue.